core/src/main/java/org/apache/iceberg/SerializableTable.java

```diff
@@ -388,6 +388,12 @@ public Transaction newTransaction() {
     throw new UnsupportedOperationException(errorMsg("newTransaction"));
   }
 
+  @Override
+  public StaticTableOperations operations() {
+    // Never return an operations that is not static
```

Comment:
I would avoid double negative here, 'always return static table operations`.  But also, I am not sure this comment serves much that the code doesnt show, should we just remove it?

Reply:
Removed